File: java/Simple form flutter/Simpleform/app/src/main/java/com/gradproject/simpleform/MainActivity.java
package com.gradproject.simpleform;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.content.DialogInterface;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button alertdialogbutton;
    private AlertDialog.Builder alertDialog;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        alertdialogbutton = findViewById(R.id.mysum);
        alertdialogbutton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                alertDialog = new AlertDialog.Builder(MainActivity.this);
                alertDialog.setTitle("Well Done");
                alertDialog.setMessage("You've entered data successfully!");
                AlertDialog dialog = alertDialog.create();
                dialog.show();
            }
        });
    }
}

File: Kotlin apps with phone functionalities/PhonecallAPP/app/src/main/java/com/badawy5/phonecallapp/MainActivity.kt
package com.badawy5.phonecallapp

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        title = "KotlinApp"

        call_button.setOnClickListener {

            val intent = Intent(Intent.ACTION_DIAL, Uri.fromParts("tel", "8344814819", null))
            startActivity(intent)

        }


    }

}